documentation/src/main/markdown/downloadingandinstalling/selfupdatingdetect.md

```diff
@@ -1,13 +1,24 @@
 # [solution_name] Version Management
 
-[solution_name] self updating feature will allow customers who choose to enable Centralized [solution_name] Version Management in [blackduck_product_name] to automate the update of [solution_name] across their pipelines.
+[solution_name] self-updating feature will allow customers who choose to enable Centralized [solution_name] Version Management in [blackduck_product_name] to automate the update of [solution_name] across their pipelines.
 
 ## Self updating [solution_name] scenarios
 
-The Self Update feature will call the `/api/tools/detect` API to check for the existence of a mapped [solution_name] version in [blackduck_product_name]. If a version that is eligible for upgrade or downgrade has been mapped, the API will redirect the request to download that version and the current execution of [solution_name] will invoke the downloaded version to execute the requested scan. 
+The Self Update feature will call the `/api/tools/detect` API end point to check for the existence of a mapped [solution_name] version in [blackduck_product_name] under "Admin > System Settings > [solution_name]". If a version that is eligible for upgrade or downgrade has been mapped, the API will redirect the request to download that version and the current execution of [solution_name] will invoke the downloaded version to execute the requested scan. 
```

Comment:
Should it be 'has been mapped the Admin'?

Reply:
Tweak coming.